src/app/preleveurs/[id]/page.js

```diff
@@ -0,0 +1,49 @@
+import {Box, Chip, Typography} from '@mui/material'
+import Link from 'next/link'
+
+import {getBeneficiaire, getPointsFromBeneficiaire} from '@/app/api/points-prelevement.js'
+import {getUsagesColors} from '@/components/map/legend-colors.js'
+
+const Page = async ({params}) => {
+  const {id} = await params
+  const preleveur = await getBeneficiaire(id)
+  const points = await getPointsFromBeneficiaire(id)
+
+  return (
+    <Box className='fr-container h-full w-full flex flex-col gap-5 mb-5'>
+      <Typography variant='h4' className='fr-mt-3w'>
+        {id} - {preleveur.civilite} {preleveur.nom} {preleveur.prenom} {preleveur.sigle} {preleveur.raison_sociale}
```

Comment:
Si l'`id` est connu des instructeurs, why not, mais sinon je ne vois pas trop l'intérêt de l'afficher.